roles/development/handlers/main.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Reload nginx
+  become: yes
+  service: name=nginx state=reloaded enabled=yes
+
+- name: Restart nginx
+  become: yes
+  service: name=nginx state=restarted enabled=yes
+
+- name: Restart mysql
+  become: yes
+  service: name=mysql state=restarted enabled=yes
```

Comment:
@wakawakaxin 
It is recommended, writing the task of ansible in multiple line with separated parameters